AmplifyPlugins/Predictions/AWSPredictionsPlugin/Dependency/AWSRekognitionAdapter.swift

```diff
@@ -0,0 +1,58 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import AWSRekognition
+
+class AWSRekognitionAdapter: AWSRekognitionBehavior {
+    let awsRekognition: RekognitionClient
```

Comment:
You get the idea 😆

Reply:
Addressed in [0e8078c](https://github.com/aws-amplify/amplify-swift/pull/2895/commits/0e8078c01c74b6070d7249a96baecf69690e19d0)